### PR TITLE
balloon_check.balloon_repeat: new case: 10000 times memory enlarge and evict

### PR DESCRIPTION
**Pull request by ybduan**
new case: 10000 times memory enlarge and evict

ID: 1467183

Signed-off-by: yduan <yduan@redhat.com>

### CODE REVIEW

**Comment by ybduan** [comment]
(1/1) smp_4.2632m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.balloon_check.balloon_repeat.balloon_base: PASS (194.53 s)

File: qemu/tests/balloon_check.py [hunk 1]
```diff
@@ -401,28 +401,30 @@ def run(test, params, env):
...
+                balloon_type = params_tag['balloon_type']
+                min_sz, max_sz = balloon_test.get_memory_boundary(balloon_type)
+                expect_mem = int(random.uniform(min_sz, max_sz))
 
-        quit_after_test = balloon_test.run_ballooning_test(expect_mem, tag)
-        if quit_after_test:
-            return
+            quit_after_test = balloon_test.run_ballooning_test(expect_mem, tag)
+            if quit_after_test:
+                return
```
Comment (yanan-fu) [inline qemu/tests/balloon_check.py]: tab or blank here ?

**Comment by aliang123** [comment]
A duplicated case with "balloon_stress", can you check it?

File: qemu/tests/balloon_check.py [hunk 2]
```diff
@@ -401,28 +401,30 @@ def run(test, params, env):
...
     try:
         balloon_test.reset_memory()
```
Comment (yanan-fu) [inline qemu/tests/balloon_check.py]: I am puzzled if we need to reset memory every time ?

File: qemu/tests/cfg/balloon_check.cfg [hunk 3]
```diff
@@ -96,3 +96,9 @@
             services_up_timeout = 30
             sub_test_after_balloon_enlarge_to_0.75 = "guest_suspend"
             # s3 s4 related params will configure in guest-os.cfg
+        - balloon_repeat:
+            only balloon_base
+            repeat_times = 5000
```
Comment (yanan-fu) [inline qemu/tests/cfg/balloon_check.cfg]: This value is  inconsistent with commit message and description, please keep consistent.

**Comment by ybduan** [comment]
After talking with @aliang123 and @huangyum , I will close this request and move this case to balloon_stress.

**Pull request closed without merging**